release_checklist.md

```diff
@@ -0,0 +1,37 @@
+# Release Checklist
+This file provides a guideline on steps that need to be taken to properly release a new version of Terrascan:
+
+### Run unit & integration tests
+The following command will kick off both unit tests and end-to-end tests. These tests should finish clean before a release is made:
+
+```
+make test
```

Comment:
```suggestion
make cicd
```

Reply:
this will also test the docker build.

Reply:
And in future we can add subtargets like `acceptance tests` and `e2e-tests` under the make `cicd` target